Zend/zend_language_parser.y

```diff
@@ -1223,7 +1223,7 @@ inline_function:
 		T_DOUBLE_ARROW backup_fn_flags backup_lex_pos expr backup_fn_flags
 			{ $$ = zend_ast_create_decl(ZEND_AST_ARROW_FUNC, $2 | $12, $1, $3,
 				  zend_string_init("{closure}", sizeof("{closure}") - 1, 0), $5, NULL,
-				  zend_ast_create(ZEND_AST_RETURN, $11), $7, NULL);
+				  $11, $7, NULL);
```

Comment:
I don't think the line break before makes sense anymore (at least the line should be indented).